### ISSUE TITLE
Reply of captured embedded youtube videos only working in Safari browser

### ISSUE THREAD

**Issue by tom-s-ukgwa** [webrecorder/webrecorder#711]
Running a local installation of webrecorder captures embedded YouTube content in html pages, using the Chrome browser, running the latest Ubuntu OS.
Capture is then verified by replaying the WARC file in webrecorder-player.
Deployment to main archive, which runs pywb leads to content only being available through the Safari browser.
In all other browsers an error message reading "An error occurred. Please try again later. (Playback ID: dozsH0G_IX8aJpUH)" is given.
The following is the YouTube embedded player debug info:
{
  "ns": "yt",
  "el": "embedded",
  "cpn": "dozsH0G_IX8aJpUH",
  "docid": "8VyX2ZgN3Ac",
  "ver": 2,
  "cmt": "0",
  "ei": "5_OcXMjcBJyTmLAPuYeWkAc",
  "fs": "0",
  "rt": "50.934",
  "euri": "https://brexitdealexplained.campaign.gov.uk/",
  "lact": 1,
  "cl": "239515163",
  "mos": 0,
  "state": "80",
  "volume": 100,
  "c": "WEB_EMBEDDED_PLAYER",
  "cver": "20190321",
  "cplayer": "UNIPLAYER",
  "cbr": "Chrome",
  "cbrver": "73.0.3683.86",
  "cos": "X11",
  "autoplay": "1",
  "hl": "en_GB",
  "cr": "GB",
  "len": "71",
  "fexp": "23707728,23710476,23736684,23741684,23744176,23749580,23751729,23751767,23752869,23755886,23755898,23760558,23762649,23777630,23780519,23785333,23794287,23794404,23794861,23796355,23797625,23798241,23798786,23799361,23799905,23800731,23801028,23801784,23803196,23804059,9405961,9449243,9452649,9486214",
  "size": "522:930",
  "inview": "1",
  "vct": "0.000",
  "vd": "NaN",
  "vpl": "",
  "vbu": "",
  "vpa": "1",
  "vsk": "0",
  "ven": "0",
  "vpr": "1",
  "vrs": "0",
  "vns": "0",
  "vec": "null",
  "vvol": "1",
  "debug_error": {
    "errorCode": "manifest.net.retryexhausted",
    "errorDetail": "backend.gvi;rc.0;rt.0.703",
    "message": "An error has occurred. Please try again later.",
    "messageKey": "GENERIC_WITHOUT_LINK",
    "subreason": ""
  },
  "relative_loudness": "NaN",
  "user_qual": "auto",
  "debug_videoId": "8VyX2ZgN3Ac",
  "0sz": false,
  "op": "",
  "yof": false,
  "dis": "",
  "gpu": "ANGLE_(Intel(R)_HD_Graphics_520_Direct3D11_vs_5_0_ps_5_0)",
  "cgr": true,
  "debug_playbackQuality": "unknown",
  "debug_date": "Thu Mar 28 2019 16:19:37 GMT+0000 (Greenwich Mean Time)"
}
Has this problem been noted previously and are there any ways to resolve it?
Many thanks,
Tom

**Comment by ikreymer** [comment]
To clarify, you're capturing with Webrecorder using Chrome and the issue is happening when replaying with just pywb using any browser besides Safari? Does it replay correctly using Webrecorder and Chrome?
Would you be willing to send us the WARC file, either here or to support@webrecorder.io?

**Comment by tom-s-ukgwa** [comment]
Yes, that's right - the WARC was created using webrecorder running on Chrome, however it now only replays in Safari (not even in Chrome).
It does also seem to be an issue with Vimeo content, in other crawls we've done.
I'll email you a link to the WARC file - it's over 160MB. 
Thanks
